### DISCUSSION TITLE
Creating Web Components

### DISCUSSION THREAD

**Discussion by senthilveln** [cdmoro/bootstrap-vue-3#343] [General]
Hi,

I had gone through the bootstrap-vue-3 components and it looks great. I have a quick question.

I am planning to create framework agnostic Web components using these components, could you please confirm if these bootstrap-vue-3 components supports web component creation.

**Comment by micene09** [comment]
As far I know, it's possible to build Vue components as web component.
This is a Vue official feature, so I think that what you are asking is just built-in, theorically speaking...it's just another build target.
